src/lib/asyncItemIterator.ts

```diff
@@ -127,6 +134,15 @@ async function process(
       return new RegExp(cli.flags.branch as string).test(pr.head.ref);
     });
   }
+  if (cli.flags.label) {
```

Comment:
Non blocker:

I presume the behavior is confusing when you specify both branch and label.

Reply:
I think today we'd apply both filters, so maybe okay? might be worth a test specifically for filter behavior.

Created issue to track:

https://github.com/googleapis/github-repo-automation/issues/525